notebooks/215-image-inpainting/215-image-inpainting.ipynb

```diff
@@ -35,7 +38,7 @@
    "metadata": {},
    "source": [
     "### Download the Model\n",
-    "Models can be downloaded with `omz_downloader` - a command-line tool for downloading models from the Open Model Zoo. `gmcnn-places2-tf` is the omz name for the considered model. You can find the names of all available models [here](https://github.com/openvinotoolkit/open_model_zoo/blob/master/models/public/index.md) and [here](https://github.com/openvinotoolkit/open_model_zoo/blob/master/models/intel/index.md). The selected model comes from the public directory, which means it must be converted into Intermediate Representation (IR). This step is skipped if the model is already downloaded.\n"
+    "Donwload model with omz name `gmcnn-places2-tf`(this step will be skipped if the model is already downloaded) and then unzip it. \n"
```

Comment:
```suggestion
    "Download `gmcnn-places2-tf`model (this step will be skipped if the model is already downloaded) and then unzip it. Downloaded model stored in TensorFlow frozen graph format. The steps how this frozen graph can be obtained from original model checkpoint can be found in this [instruction](https://docs.openvino.ai/latest/omz_models_model_gmcnn_places2_tf.html#steps-to-reproduce-conversion-to-frozen-graph)\n"
```
and please also add link to instruction how to convert original checkpoint to frozen graph ()

Reply:
done